src/Core/src/Platform/Windows/TextBoxExtensions.cs

```diff
@@ -78,14 +78,6 @@ public static void UpdatePlaceholderColor(this MauiTextBox textBox, IEditor edit
 		public static void UpdateFont(this MauiTextBox nativeControl, IText text, IFontManager fontManager) =>
 			nativeControl.UpdateFont(text.Font, fontManager);
 
-		public static void UpdateFont(this MauiTextBox nativeControl, Font font, IFontManager fontManager)
```

Comment:
This is just duplicated code. There's already an extension method that takes "TextBox" with the same code